mod/edit/set/all/form_buttons.rb

```diff
@@ -3,6 +3,19 @@ def standard_submit_button
     output [standard_save_button, standard_save_and_close_button]
   end
 
+# Generates a standard save button with optional parameters.
+#
+# @param [Hash] opts The options for the save button.
+# @option opts [String] :text ("Save") The text displayed on the button.
+# @option opts [String] :data-cy ("save") The value for the 'data-cy' attribute
+#   used for testing or identification purposes.
+#
+# @return [String] The HTML code for the standard save button.
+#
+# @example
+#   standard_save_button(text: "Update")
+#   #=> "<button class='submit-button me-3 _update-history-pills' data-cy='save'\
+#   data-disable-with='Submitting'>Update</button>"
```

Comment:
here's a good example of why long explicit examples cause problems. If someone changes the default "disable-with" value in `submit_button`, it would be very hard for them to realize that some other documentation somewhere assumed it wouldn't change.

Fragile documentation is even worse than no documentation, because it ends up being incorrect and therefore worse than having to read the code.